src/BloomExe/Edit/WebThumbNailList.cs

```diff
@@ -615,6 +630,14 @@ private List<IPage> UpdateItems(IEnumerable<IPage> pages)
 				var captionOrPageNumber = page.GetCaptionOrPageNumber(ref pageNumber);
 				captionDiv.InnerText = LocalizationManager.GetDynamicString("Bloom", "EditTab.ThumbnailCaptions." + captionOrPageNumber, captionOrPageNumber);
 			}
+
+			// set interval based on physical RAM
+			var hidden = pageDoc.CreateElement("input");
```

Comment:
Seems curious to use an input element to hold this. Is this perhaps a holdover from when you were experimenting and wanted to change the value easily?  Could you instead add a meta element, or perhaps a "data-thumbnail-interval" to, say, the body of the dom?